src/dracula.yml

```diff
@@ -39,6 +39,7 @@ dracula:
   other:
     - &COMMENT       '#6272A4'
     - &LineHighlight '#44475A75'
+    - &Whitespace    '#44475AC8'
```

Comment:
Can we try `#424450` for whitespace and see what that looks like?

Reason being: I'd like to keep as closely uniform as I can to [dracula/vim](https://github.com/dracula/vim/blob/master/colors/dracula.vim#L42). 

Also, the `C8` part to my knowledge shouldn't do anything here. The extended 2 digits of the hex values accounts for alpha and that needs to be `00-99` inclusive.

Reply:
Also, can we call this `NonText` rather than `Whitespace` to stay uniform with the vim terminology?